pkg/test/fakes/policy/backend.go

```diff
@@ -224,11 +237,44 @@ func (b *Backend) HandleCheckNothing(ctx context.Context, req *checknothing.Hand
 	}
 
 	scope.Infof("Backend.HandleCheckNothing => OK")
+	validDuration := 5 * time.Second
```

Comment:
Nit: You can make these configurable. PolicyBackend has a control connection for controlling its behavior.

Reply:
Done.